package-lock.json

```diff
@@ -1299,7 +1299,10 @@
       "requires": {
         "@lerna/query-graph": "file:utils/query-graph",
         "figgy-pudding": "^3.5.1",
-        "p-queue": "^4.0.0"
+        "fs-extra": "^8.1.0",
+        "npmlog": "^4.1.2",
+        "p-queue": "^6.2.1",
```

Comment:
Unfortunately, p-queue 5+ does not support node v6, and thus this is a breaking change. Any specific reason this needed to be upgraded, anyway?

(I plan on dropping support for node v6 + v8 real soon now, but I'd rather avoid it in this PR.

Reply:
No, I can revert it. I was playing with the new custom QueueClass which was introduced in p-queue 6 in an earlier iteration of the PR. When I abandoned that strategy, I just kept the upgrade.